pkg/registry/generic/options.go

```diff
@@ -16,7 +16,11 @@ limitations under the License.
 
 package generic
 
-import "k8s.io/kubernetes/pkg/storage/storagebackend"
+import (
+	"k8s.io/kubernetes/pkg/genericapiserver"
```

Comment:
A dependency from pkg/registry back to master is wrong as well. I have moved the two interfaces now to pkg/registry. I think that's the right place. @deads2k ptal